base/docs/helpdb/Base.jl

```diff
@@ -2350,6 +2450,33 @@ countlines
     .\\(x, y)
 
 Element-wise left division operator.
+
+```jldoctest
+julia> A = [1 0; 0 -1];
```

Comment:
@tkelman didn't like my simple examples for `.+` and `.*` so I'm going all out. I think this is a good example to show people some cool things Julia can do. If they get too confused they can go to `broadcast` and learn so many wonderful things.